### PR TITLE
Edit weather vignette

### PR DESCRIPTION
**Pull request by KristinaRiemer**
To address everything in #102, plus a few extra things.

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
If no one has any concerns about this, I'm going to merge it tomorrow morning.

File: vignettes/02-get-weather-data-R.Rmd [hunk 1]
```diff
@@ -117,34 +47,26 @@ daily_values = weather_data %>%
...
-
-### Get all available weather variables
+We can also plot the time series for all eight of the weather variables in a single figure. 
+We first have to rearrange the data to making plotting possible using R package `ggplot`. 
 
-The weather variables that are available from these datapoints data are 
-extracted below from the column names of the dataframe that we read in earlier. 
-Any of these variables that are of interest can be analyzed and plotted. 
+```{r}
+weather_data_long = weather_data %>% select(-source, -source_file) %>% gather(weather_variable, observation, -time)
```
Comment (jeanetteclark) [inline vignettes/02-get-weather-data-R.Rmd]: I'd break up this statement between the pipes (consistent with your met-GDD chunk)

File: vignettes/02-get-weather-data-R.Rmd [hunk 2]
```diff
@@ -117,34 +47,26 @@ daily_values = weather_data %>%
                          (max_temp + min_temp) / 2 - 10, 0))
 ```
 
-### Plot data using R
+## Plot data using R
```
Comment (jeanetteclark) [inline vignettes/02-get-weather-data-R.Rmd]: Yay! Nice headers

File: vignettes/02-get-weather-data-R.Rmd [hunk 3]
```diff
@@ -1,110 +1,40 @@
...
+This vignette shows how to read weather data for the month of January 2017 from the [weather station](https://cals-mac.arizona.edu/weather-station) at the University of Arizona's [Maricopa Agricultural Center](http://cals-mac.arizona.edu/) into R. 
+These data are stored online on the data management system Clowder, which is accessed using an API. 
+More detailed information about the structure of the database and how API URLs are created is available in the [weather tutorial]("sensors/01-meteorological-data.Rmd"). 
+Data across time for one weather variable, temperature, is plotted in R. 
+Then all eight of the weather variables have their times series plotted. 
 
-### Using the API to get data
+## Read in data using R
 
-In order to access the data, we need to contruct a URL that links to where the
```
Comment (jeanetteclark) [inline vignettes/02-get-weather-data-R.Rmd]: I like moving this section 👍

**Comment by jeanetteclark** [comment]
This looks good! I would restructure the first section (objectives and introduction, see my PR), but it is obviously up to the group to decide what structure to adopt

File: sensors/01-meteorological-data.Rmd [hunk 4]
```diff
@@ -48,31 +48,53 @@ We want to standardize the units as well as part of the `met2CF.<product>` step.
...
+### Using the API to get data
 
-![Schema](https://cloud.githubusercontent.com/assets/9286213/16991300/b2f2b09a-4e60-11e6-96b7-8b63c3d1f995.jpg)
+In order to access the data, we need to contruct a URL that links to where the 
+data is located on [Clowder](https://clowder.ncsa.illinois.edu/). The data is 
+then pulled down using the API, which ["receives requests and sends responses"](https://medium.freecodecamp.org/what-is-an-api-in-english-please-b880a3214a82)
+, for Clowder. 
 
-#### Querying the API
+### The structure of the database
```
Comment (dlebauer) [inline sensors/01-meteorological-data.Rmd]:
```suggestion
### The structure of the Geostreams database
```

File: sensors/01-meteorological-data.Rmd [hunk 5]
```diff
@@ -48,31 +48,53 @@ We want to standardize the units as well as part of the `met2CF.<product>` step.
 
 The key is to process each type of met data (site, reanalysis, forecast, climate scenario, etc) to the exact same standard. This way every operation after that (extract, gap fill, downscale, convert to a model, etc) will always have the exact same inputs. This will make everything else much simpler to code and allow us to avoid a lot of unnecessary data checking, tests, etc being repeated in every downstream function.
 
-### The Geostreams Database
+### Using the API to get data
 
-![Schema](https://cloud.githubusercontent.com/assets/9286213/16991300/b2f2b09a-4e60-11e6-96b7-8b63c3d1f995.jpg)
+In order to access the data, we need to contruct a URL that links to where the 
+data is located on [Clowder](https://clowder.ncsa.illinois.edu/). The data is
```
Comment (dlebauer) [inline sensors/01-meteorological-data.Rmd]:
```suggestion
data is located on [Clowder](https://terraref.ncsa.illinois.edu/clowder). The data is 
```

File: vignettes/02-get-weather-data-R.Rmd [hunk 6]
```diff
@@ -117,34 +47,28 @@ daily_values = weather_data %>%
...
-
-### Get all available weather variables
+We can also plot the time series for all eight of the weather variables in a single figure. 
+We first have to rearrange the data to making plotting possible using R package `ggplot`. 
 
-The weather variables that are available from these datapoints data are 
-extracted below from the column names of the dataframe that we read in earlier. 
-Any of these variables that are of interest can be analyzed and plotted. 
+```{r}
+weather_data_long = weather_data %>%
```
Comment (dlebauer) [inline vignettes/02-get-weather-data-R.Rmd]:
```suggestion
weather_data_long <- weather_data %>% 
```

**Pull request merged**